### PR TITLE
These are my changes

### PR DESCRIPTION
**Pull request by JohannaBrown**
I changed the about.jsp to include the mini profiles of the team members of CodeU Team 32! The other files are not different from how the originals were.

### CODE REVIEW

**Comment by JohannaBrown** [comment]
I added myself as a reviewer lol

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   limitations under the License.
 -->
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-    <application>YOUR_PROJECT_ID</application>
```
Comment (JohannaBrown) [inline src/main/webapp/WEB-INF/appengine-web.xml]: Uploaded the wrong appe engine xml

Comment (jess2422) [reply src/main/webapp/WEB-INF/appengine-web.xml]: Good catch. Do you know what the diff is between the two appengine-web.xml docs?

Comment (UnitingDust) [reply src/main/webapp/WEB-INF/appengine-web.xml]: The application tag should be our own unique "project id" right?

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
+    <application>chat-web-applet-team-32</application>
```
Comment (JohannaBrown) [inline src/main/webapp/WEB-INF/appengine-web.xml]: This line has the correct application tag @UnitingDust

File: src/main/webapp/WEB-INF/appengine-web.xml (hunk 1 above)
Comment (JohannaBrown) [reply src/main/webapp/WEB-INF/appengine-web.xml]: The difference should be the project id? @jess2422

File: src/main/webapp/WEB-INF/appengine-web.xml (hunk 2 above)
Comment (UnitingDust) [reply src/main/webapp/WEB-INF/appengine-web.xml]: Got it!

**Pull request merged**